src/LuckyBuy.sol

```diff
@@ -73,17 +73,21 @@ contract LuckyBuy is
     event CosignerAdded(address indexed cosigner);
     event CosignerRemoved(address indexed cosigner);
     event Fulfillment(
-        address indexed sender,
-        uint256 indexed commitId,
+        bytes32 indexed digest,
+        uint256 commitId,
+        address receiver,
```

Comment:
nit: It might be useful to index `receiver` so events can be filtered by LB player more simply.